configs/tasks/obj_nav_mp3d.yaml

```diff
@@ -0,0 +1,47 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 750
+SIMULATOR:
+  TURN_ANGLE: 30
+  TILT_ANGLE: 30
+  AGENT_0:
+    SENSORS: ['RGB_SENSOR', 'DEPTH_SENSOR']
+    HEIGHT: 0.88
+    RADIUS: 0.18
+  HABITAT_SIM_V0:
+    GPU_DEVICE_ID: 0
+  SEMANTIC_SENSOR:
+    WIDTH: 640
+    HEIGHT: 480
+    HFOV: 59
```

Comment:
Where is the HFOV and width/height coming from?

Reply:
> Where is the HFOV and width/height coming from?

From suggests from AI2 folks (@roozbehm and Ani).

Reply:
Resolution is coming from PyRobot config, HFOV coming from the AI team.

Reply:
Ah, they are using a square aspect ratio, so we should set this to 59*4/3 to be consistent (we are using 4:3 and this is hfov).